### PR TITLE
CMS ARS 3.1 Low Overlay

### PR DESCRIPTION
**Pull request by karikarshivani**

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -17,30 +17,25 @@ inputs:
     description: 'The username MSSQL DB Server'
```
Comment (karikarshivani) [inline inspec.yml:17]: I'll update the branch name in the `depends.url` when the PR for the baseline has been approved and merged. Thanks!

**Review by ejaronne — CHANGES_REQUESTED** [review]
microsoft-sql-server-2017-cis-1.2 relates to SI-10 (only for Mod/High) and SC-32 (non-mandatory). In planning in CFACTS, CMS ARS 3.1 non-mandatory controls are optional for ISSOs to adopt, and therefore may be something they still wish to include in their application's control set. Therefore, we don't exclude (make not applicable) a requirement based on non-mandatory NIST controls.

**Comment by ejaronne** [comment]
update README exec instructions to use main.tar.gz (not master)
update README "full heimdall server" url to point to heimdall2 repo not old heimdall

**Review by ejaronne — APPROVED** [review]

**Comment by karikarshivani** [comment]
> microsoft-sql-server-2017-cis-1.2 relates to SI-10 (only for Mod/High) and SC-32 (non-mandatory). In planning in CFACTS, CMS ARS 3.1 non-mandatory controls are optional for ISSOs to adopt, and therefore may be something they still wish to include in their application's control set. Therefore, we don't exclude (make not applicable) a requirement based on non-mandatory NIST controls.

Good catch! Removed the caveat from cis-1.2
PR ready to be merged. Thanks!

**Pull request merged**